src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -763,28 +763,32 @@
 									Grid.Row="0"
 									Grid.Column="0"
 									VerticalAlignment="Center"
-									Text="{x:Bind Commands.ToggleShowHiddenItems.Label}" />
+									Text="{x:Bind Commands.ToggleShowHiddenItems.Label}"
+									ToolTipService.ToolTip="{x:Bind Commands.ToggleShowHiddenItems.LabelWithHotKey, Mode=OneWay}" />
```

Comment:
TextBlocks don't normally get tooltips, is it really needed here?

Reply:
I removed the tooltip on the label. The tooltip on the button is just the hotkey.